### ISSUE TITLE
formula simplication

### ISSUE THREAD

**Issue by yxliang01** [pysathq/pysat#26]
Would be great if pysat supports CNF formula simplification :)

Thanks

**Comment by alexeyignatiev** [comment]
Hi @yxliang01, I agree that it would be great. This feature would help a lot in many use cases. Therefore, PRs are welcome! :) Otherwise, I will take a look into this only when I have time. :)

**Comment by yxliang01** [comment]
@alexeyignatiev Yup! It definitely be very useful. However, I also can't allocate time for this recently... You have an estimate for this? It would be super helpful for my research projects if this is implemented! :) (PySat already facilitated my project, thanks for that)

**Comment by alexeyignatiev** [comment]
Unfortunately, I cannot estimate when this could be done. BTW, what kind of simplification do you mean? Two-level minimization or formula preprocessing?

**Comment by yxliang01** [comment]
I meant simply formula preprocessing. E.g. removing clauses that always hold, etc...
